src/request/components/request-detail-panel-execution.jsx

```diff
@@ -145,11 +272,59 @@ var CommandList = React.createClass({
                     ); 
             }
         }
+            
+        if (mongoDBMessages ) {
+            var mongoOperations = [];
+            for (var i = 0; i < mongoDBMessages.length; i++) {
+                // TODO:  figure out why end-request ordinal is greater than mongo ordinal. 
+                if (mongoDBMessages[i].ordinal > beginMessage.ordinal /*&& mongoDBMessages[i].ordinal < endMessage.ordinal*/) {
+                    var commandItem = CreateMongoCommandItem(mongoDBMessages[i]);
+                    if ( commandItem ) {
+                        mongoOperations.push(commandItem);
+                     }
+                }
+         }
+            
+         // process action
+         if (mongoOperations.length > 0) {
+                content = (
+                        <div className="tab-section tab-section-execution-command">
+                            <div className="flex flex-row flex-inherit tab-section-header">
+                                <div className="tab-title col-9">MongoDB</div>
+                            </div>
+                            <div className="tab-section-boxing">
+                                <section className="flex flex-row flex-inherit flex-base tab-section-item">
+                                    <div className="tab-section-execution-command-items col-9">{mongoOperations}</div>
+                                </section>
+                            </div>
+                        </div>
+                    ); 
+            }
+        }
         
         return content;
     }
 });
 
+function CreateMongoCommandItem(mongoDBMessage) {
+    // TODO:  should be a more efficient way to do the below, particularly since the mongo messages are a
+    //        already sorted by message type.
+    var item;
+    if (_.intersection(mongoDBMessage.types, ['data-mongodb-insert']).length === 1){
+        item = <MongoInsertCommand key={mongoDBMessage.id} mongoOperation={mongoDBMessage} />
+    }
+    else if (_.intersection(mongoDBMessage.types, ['data-mongodb-read']).length === 1){
+       item = <MongoReadCommand key={mongoDBMessage.id} mongoOperation={mongoDBMessage} />
+    }
+    else if (_.intersection(mongoDBMessage.types, ['data-mongodb-update']).length === 1){
+       item = <MongoUpdateCommand key={mongoDBMessage.id} mongoOperation={mongoDBMessage} />  
+    }
+    else if (_.intersection(mongoDBMessage.types, ['data-mongodb-delete']).length === 1){
+       item = <MongoDeleteCommand key={mongoDBMessage.id} mongoOperation={mongoDBMessage} />
+    }
+    return item;
+}
```

Comment:
- `intersection` - We have some helpers which might make this logic a bit easier... https://github.com/Glimpse/Glimpse.Client.Prototype/blob/master/src/request/util/request-message-processor.js#L25-L56. It also turns out that we have the data sliced and diced in a few different ways, so hopefully that might simplify some of the logic here.
 - Next iteration - from here we should be able to do the above based off a list of registered components that are bound to a given message type. This means that we slide SQL Commands, etc into the mix more easily.